---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I'm a PhD candidate in the <a href="http://www.imperial.ac.uk/immunology-inflammation/">Department of Immunology and Inflammation</a> at Imperial College London. I'm interested in data-driven research and the -Omics technologies. 

My core project involves elucidating the role of genes and proteins in the context of inflammatory disease. I am currently combining genomic, <a href="https://elifesciences.org/articles/64827">proteomic</a> and RNA sequencing data to identify molecular pathways that play a driving role in the pathogenesis of COVID-19. These datasets involved end-stage kidney disease (ESKD) patients (n>300). ESKD patients are unable to shield as they must come to hospital approximately 3 times a week; this has presented a unique opportunity for longitudinal blood sampling in both the outpatient and inpatient settings, allowing us to capture the full spectrum of disease severity and model its temporal evolution.

I am also interested in <a href="/software/">developing methods</a> for the processing and data-mining of -Omics datasets. I previously developed a pipeline for the <i>de novo</i> annotation of Pack-TYPE transposons based on sequence features. These type 2 DNA transposons are able to capture chromosomal DNA of their host and contribute to the evolution of genes. I implemented this as an R package which is <a href="https://doi.org/doi:10.18129/B9.bioc.packFinder">available</a> as part of the Bioconductor project. The development of this method led to the discovery of the widespread nature of these transposable elements in plants, revealing the extensive impact they have had on host genes. I am also involved in the development of <a href="https://github.com/computational-metabolomics/metaboblend">Metaboblend</a>, a Python package for the <i>de novo</i> structural elucidation of small molecules in mass spectrometry-based Metabolomics. Metabolite annotation is a major bottleneck for untargeted metabolomics studies.
